### PR TITLE
Split max_distance into max_sniping_distance and max_walking_distance

### PR DESCRIPTION
**Pull request by Jasperrr91**
## Short Description:
Split max_distance into max_sniping_distance and max_walking_distance

## Fixes/Resolves/Closes (please use correct syntax):
- as discussed in https://github.com/PokemonGoF/PokemonGo-Bot/pull/4969

### CODE REVIEW

File: docs/configuration_files.md [hunk 1]
```diff
@@ -634,8 +635,10 @@ This task will fetch current pokemon spawns from /raw_data of an PokemonGo-Map i
     "type": "MoveToMapPokemon",
     "config": {
       "address": "http://localhost:5000",
-      "//NOTE: Change the max_distance to adjust the max sniping range (km)": {},
-      "max_distance": 500,
+      "//NOTE: Change the max_sniping_distance to adjust the max sniping range (m)": {},
+      "max_sniping distance": 10000,
+      "//NOTE: Change the max_walking_distance to adjust the max walking range when snipe is off (m)": {},
+      "max__walking_distance": 500,
```
Comment (solderzzc) [inline docs/configuration_files.md]: Typo here...

File: docs/configuration_files.md [hunk 2]
```diff
@@ -634,8 +635,10 @@ This task will fetch current pokemon spawns from /raw_data of an PokemonGo-Map i
     "type": "MoveToMapPokemon",
     "config": {
       "address": "http://localhost:5000",
-      "//NOTE: Change the max_distance to adjust the max sniping range (km)": {},
-      "max_distance": 500,
+      "//NOTE: Change the max_sniping_distance to adjust the max sniping range (m)": {},
```
Comment (solderzzc) [inline docs/configuration_files.md]: is that better to use `km` in sniper distance?

**Comment by solderzzc** [comment]
👍

File: docs/configuration_files.md (hunk 2 above)
Comment (Jasperrr91) [inline docs/configuration_files.md]: Yea I think that's much more readable. I'll add that.

File: docs/configuration_files.md (hunk 1 above)
Comment (Jasperrr91) [inline docs/configuration_files.md]: Ahh it was late, sorry.

**Comment by mjmadsen** [comment]
We need to throw a warning that max_distance is depreciated. Lots of folks not updating their config.

**Pull request merged**